code/modules/projectiles/guns/projectile/shotgun.dm

```diff
@@ -48,6 +49,7 @@
 	icon = 'icons/obj/guns/shotgun.dmi'
 	icon_state = "shotgun"
 	item_state = "shotgun"
+	accuracy = 0
```

Comment:
why not 1 if the parent is? it's not really different from any of the other shotguns.

Reply:
this was copied from the PR i ported the code from, not sure why. should i remove it?